components/cpm/Analysis.vue

```diff
@@ -2,50 +2,107 @@
   <div class="w-full h-full">
     <span class="space-x-3 p-3">
       <CpmDropdown v-model="selectedVariable" :options="variables" alttext="Choose a variable." />
-      <CpmDropdown v-model="selectedResolution" :options="resolution" alttext="Select a resolution." />
-      <CpmDropdown v-model="selectedCategory" :options="categories" alttext="Choose a category" />
-      <CpmDropdown v-model="selectedDate" :options="dates" alttext="Select a date." />
+      <CpmDropdown v-model="selectedSeason" :options="seasons" alttext="Select a season." />
+      <CpmDropdown v-model="selectedModel" :options="models" alttext="Choose a model/project" />
+      <!-- same style but not selectable option to display region -->
+      <select
+        class="border border-gray-300 rounded-full cursor-pointer
+      text-gray-600 h-10 pl-5 pr-10 bg-white hover:border-gray-400
+      focus:outline-none appearance-none"
+      >
+        <option :value="domain">
+          {{ domain }}
+        </option>
+      </select>
     </span>
-    <div
-      class="bg-no-repeat bg-left-top bg-contain w-full h-full"
-      :style="{backgroundImage: `url(${bgImage})` }"
-    />
+    <div class="w-full h-full flex flex-wrap">
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${cpmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          CPM experiment
```

Comment:
Thanks for noticing this! I only tested this on my wider screen...That's why. It is good to check everything on smaller screen as well. Indeed we have the same issue for Atlas since the image (map) is displayed as the background image. It can be solved by separating the title and the map `div`, though it will take more space. But it works well now.